osmosis-types/gamm/pool-models/balancer/marshal.go

```diff
@@ -2,69 +2,65 @@ package balancer
 
 import (
 	"encoding/json"
-	"errors"
-	"strings"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-type balancerPoolPretty struct {
-	Address            sdk.AccAddress `json:"address" yaml:"address"`
-	Id                 uint64         `json:"id" yaml:"id"`
-	PoolParams         PoolParams     `json:"pool_params" yaml:"pool_params"`
-	FuturePoolGovernor string         `json:"future_pool_governor" yaml:"future_pool_governor"`
-	TotalWeight        sdk.Dec        `json:"total_weight" yaml:"total_weight"`
-	TotalShares        sdk.Coin       `json:"total_shares" yaml:"total_shares"`
-	PoolAssets         []PoolAsset    `json:"pool_assets" yaml:"pool_assets"`
-}
+// type balancerPoolPretty struct {
```

Comment:
remove this ?/ or we need it for some ref.

Reply:
Mostly so we make sure that when we bump the osmosis-types dependency in the future we ensure we remove it. A comment will suffice though.